src/module.c

```diff
@@ -442,6 +444,29 @@ char *RM_Strdup(const char *str) {
     return zstrdup(str);
 }
 
+void RM_ProcessEventsWhileBlocked(RedisModuleCtx *ctx) {
```

Comment:
please add a documentation comment above the API.
maybe rename it to `RM_SlowContextHeartbeat` (i.e. like the existing `RM_SendChildHeartbeat`)?
i.e. something more abstract.
@guybe7 @MeirShpilraien WDYT?